modules/logging-loki-storage.adoc

```diff
@@ -0,0 +1,274 @@
+// Module is included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+:_content-type: PROCEDURE
+[id="logging-loki-storage_{context}"]
+= Loki log storage
+
+The Loki Operator supports link:https://aws.amazon.com/[AWS S3], link:https://azure.microsoft.com[Azure], link:https://cloud.google.com/[GCS], link:https://min.io/[Minio],  link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[OpenShift Data Foundation], link:https://us.alibabacloud.com/[Alibaba Cloud] and link:https://docs.openstack.org/swift/latest/[Swift] for LokiStack object storage.
```

Comment:
Does it make sense to group the "S3-compatible" storages (AWS, Minio, ODF, Alibaba-Cloud) together, so that it becomes visible that we're not explicitly supporting _these storages_ but can work with anything that has a S3-compatible API (in addition to some others like GCS which _do_ have a different API).